system/systemd/kazoo-app.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Kazoo whistle_app server daemon
+Wants=epmd.socket
+After=syslog.target network.target auditd.service epmd.socket
+
+[Service]
+Type=forking
+User=kazoo
+Group=daemon
+WorkingDirectory=/opt/kazoo
+Environment="ERL_LIBS=/opt/kazoo/deps:/opt/kazoo/core:/opt/kazoo/applications:$(echo /opt/kazoo/deps/rabbitmq_erlang_client-*/deps)"
```

Comment:
yes, there's no need since it will be part of `deps`. i'm not even sure if
this is required since the kazoo startup script will take care of this, all
apps, core & deps go into a single dir called `lib` under kazoo.


On Thu, Apr 14, 2016 at 11:07 AM, sergey-safarov <notifications@github.com>
wrote:

> In system/systemd/kazoo-app.service
> <https://github.com/2600hz/kazoo-configs/pull/76#discussion_r59691942>:
>
> > @@ -0,0 +1,25 @@
> > +[Unit]
> > +Description=Kazoo whistle_app server daemon
> > +Wants=epmd.socket
> > +After=syslog.target network.target auditd.service epmd.socket
> > +
> > +[Service]
> > +Type=forking
> > +User=kazoo
> > +Group=daemon
> > +WorkingDirectory=/opt/kazoo
> > +Environment="ERL_LIBS=/opt/kazoo/deps:/opt/kazoo/core:/opt/kazoo/applications:$(echo /opt/kazoo/deps/rabbitmq_erlang_client-*/deps)"
>
> This mean - requred to delete string "$(echo
> /opt/kazoo/deps/rabbitmq_erlang_client-*/deps)"?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/2600hz/kazoo-configs/pull/76/files/ca36faa0d18a137f02f05c1280ab90a3a1758d9f#r59691942>
>